content/docs/howto/go.md

```diff
@@ -270,6 +270,68 @@ When building with the pack CLI, create a [project.toml][cnb/project-file] file
     value="assets/*:public/*"
 {{< /code/copyable >}}
 
+## Enable Process Reloading
+By default, your Go server will be the only process running in your app
+container at runtime. You can enable restarting the server process
```

Comment:
```suggestion
By default, your Go binary will be the only process running in your app
container at runtime. You can enable restarting the binary process
```
I think that this might make more sense in the Go copy because you could potentially write services or something of that nature in Go and still want to test them locally. This is not a strongly held copy change though.